cypress/utils/makeScreenshots.ts

```diff
@@ -21,6 +21,8 @@ export function makeScreenshots(name: string, viewports: string[] = ['macbook-15
 
     for (const viewport of viewports) {
       cy.viewport(viewport as any);
+      cy.get('body').then(e => e.append('<textarea style="position: absolute; top: 0; opacity: 0" />'));
```

Comment:
i think we could move it out of the loop and do it only once. Also: what about removing it after? Maybe give it a unique ID and then it's easy to find it and remove.